code/web/src/modules/user/Profile.test.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import Profile from './Profile'
+
+import { render, fireEvent } from '@testing-library/react'
+import { BrowserRouter } from  'react-router-dom'
+import '@testing-library/jest-dom'
+
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import { rootReducer } from '../../setup/store'
+
+let store;
+
+describe('Profile', () => {
+  beforeEach(() => {
+    store = createStore(rootReducer, {
+      user: {
+        error: null,
+        isLoading: false,
+        isAuthenticated: true,
+        details: { 
+          name: 'The User',
+          email: 'user@crate.com',
+          role: 'USER'
+        }
+      }
+    })
+  })
+
+  it('should render other components', () => {
+    const { getByText, getByTestId } = render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <Profile />
+        </Provider>
+      </BrowserRouter>
+    )
+
+    const shippingAddress = getByText('Shipping Address')
+    const deliveryDate = getByText('Delivery Date', { exact: false })
```

Comment:
What does this exact:false indicate?

Reply:
That means that the text doesn't have to exactly match.  It might say 'Delivery Date 11/05' but I am just grabbing the first part of that